ui/app/styles/core/title.scss

```diff
@@ -8,4 +8,8 @@
   &.with-headroom {
     margin-top: 1rem;
   }
+
+  &.with-subheading {
+    margin-bottom: 0.5rem;
```

Comment:
Can we bump this to .85rem? The 16px I had before was probably a little too generous, but .85 gets us to 12px, which is still a standard spacing class for Structure.